x-pack/plugins/case/server/connectors/index.ts

```diff
@@ -41,3 +49,19 @@ export const externalServiceFormatters: ExternalServiceFormatterMapper = {
   '.jira': jiraExternalServiceFormatter,
   '.resilient': resilientExternalServiceFormatter,
 };
+
+export const isCommentGeneratedAlert = (
+  comment: CommentSchemaType | CommentRequest
+): comment is ContextTypeGeneratedAlertType => {
+  return (
+    comment.type === CommentType.generatedAlert &&
+    'alerts' in comment &&
```

Comment:
I think the type check is enough, right?

Reply:
Just being extra careful since we need to access the `alerts` field.